packages/material-ui/src/Accordion/Accordion.js

```diff
@@ -2,21 +2,52 @@ import * as React from 'react';
 import { isFragment } from 'react-is';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import { chainPropTypes } from '@material-ui/utils';
+import { deepmerge, chainPropTypes } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import Collapse from '../Collapse';
 import Paper from '../Paper';
-import withStyles from '../styles/withStyles';
 import AccordionContext from './AccordionContext';
 import useControlled from '../utils/useControlled';
+import accordionClasses, { getAccordionUtilityClass } from './accordionClasses';
 
-export const styles = (theme) => {
-  const transition = {
-    duration: theme.transitions.duration.shortest,
+const overridesResolver = (props, styles) => {
+  const { styleProps } = props;
+
+  return deepmerge(styles.root || {}, {
+    ...(!styleProps.square && styles.rounded),
+    ...(styleProps.expanded && styles.expanded),
+    ...(styleProps.disabled && styles.disabled),
+  });
```

Comment:
```suggestion
    [`& .${accordionClasses.region}`]: styles.region,
  });
```

We missed adding overrides for the `region` slot.